lighthouse-treemap/app/styles/treemap.css

```diff
@@ -65,6 +65,12 @@ body {
   background-color: lightblue;
 }
 
+.view-mode__button {
```

Comment:
btw you're losing your cursor styling because label has a default (who knew)

![image](https://user-images.githubusercontent.com/39191/111365472-c1639780-864f-11eb-8de1-eb96958940fb.png)